examples/evkbmimxrt1060/test/app_main.c

```diff
@@ -70,6 +68,30 @@ extern void vCLITask( void * pvParam );
 
 extern void vOTAUpdateTask( void * pvParam );
 
+int RunDeviceAdvisorDemo( void )
+{
+    BaseType_t xResult = pdFAIL;
+
+    xResult = xMQTTAgentInit( appmainMQTT_AGENT_TASK_STACK_SIZE, appmainMQTT_AGENT_TASK_PRIORITY );
+
+    if( xResult == pdPASS )
+    {
+        xResult = xTaskCreate( vSubscribePublishTestTask,
+            "TEST",
+            appmainTEST_TASK_STACK_SIZE,
+            NULL,
+            appmainTEST_TASK_PRIORITY,
+            NULL );
+
+        if (xResult == pdPASS)
+        {
+            return 0;

```

Comment:
In general the coding style is to have single return statement in all functions.